### PR TITLE
Update config structure +  blockchain conferences

### PR DESCRIPTION
**Pull request by silksil**
- Update config by putting it in seperate files
- only show blockchain conferences that are active
- Data structure of the conferences are set up in such a way, that easy to update with this notion file: https://www.notion.so/windingtree/Database-of-blockchain-conferences-63a2b13af8a647c39231f17f7756f4f6

### CODE REVIEW

File: src/config/events.ts [hunk 1]
```diff
@@ -3198,6 +3341,30 @@ const allEvents = [
     Youtube: 'https://www.youtube.com/channel/UCcTmSSIs_rRyUOlQKJZ4lpg',
     '📮': ''
   },
+  {
```
Comment (dave-ok) [inline src/config/events.ts:3344]: Why this empty event?

File: src/config/events.ts [hunk 2]
```diff
@@ -0,0 +1,3289 @@
...
+      latlong: latlong
+    };
+
+    return normalizedEvent;
+  }
+);
+export const filteredEvents: EventItemType[] = normalizedEvents.filter(
+  ({ name, location, image, startDate, endDate }) => {
+    const isInPast = endDate && getIsInPast(new Date(endDate));
+    const isToday = endDate && getIsToday(new Date(endDate));
```
[the remark this answers is not in the document]
Comment (silksil) [reply src/config/events.ts:3441]: only if the end date is today. If the startdat is today and enddate is tomorrow we won't filter it.


The reason is that the end and start date cannot be the same on the search page.

File: src/config/events.ts (hunk 1 above)
Comment (silksil) [reply src/config/events.ts:3344]: this is how we are getting it from the notion page: https://www.notion.so/Database-of-blockchain-conferences-63a2b13af8a647c39231f17f7756f4f6

This should not appear in the UI though

File: src/config/events.ts [hunk 3]
```diff
@@ -0,0 +1,3289 @@
...
+    Twitter: 'https://twitter.com/CFCstmoritz',
+    Youtube: '',
+    '📮': 'email'
+  }
+];
+
+const normalizedEvents = allEvents.map(
+  ({ latitude, longitude, name, location, image, startDate, endDate, conferenceUrl }) => {
+    const latlong =
+      latitude.trim() === '' && longitude.trim() === ''
```
[the remark this answers is not in the document]
Comment (silksil) [reply src/config/events.ts]: you are right

File: src/config/events.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (dave-ok) [reply src/config/events.ts:3441]: Cool!

File: src/config/events.ts (hunk 1 above)
Comment (dave-ok) [reply src/config/events.ts:3344]: Got it!

**Review by dave-ok — APPROVED** [review]
Awesome!

**Pull request merged**